### PR TITLE
[UT] add ut for pkg/registry/networking/ipaddress

### PR DESCRIPTION
**Pull request by taoyounger**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:

See [here](https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit&include-filter-by-regex=pkg/registry/networking/ipaddress) to find the UT code for ipaddress Coverage is 0, increase for it in this pr. :) 

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:
Does this PR introduce a user-facing change?
```
None
```

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```
None
```

### CODE REVIEW

File: pkg/registry/networking/ipaddress/strategy_test.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```
[the remark this answers is not in the document]
Comment (ty-dc) [reply pkg/registry/networking/ipaddress/strategy_test.go]: Thanks for your reply, it has been corrected.

**Comment by ty-dc** [comment]
> /ok-to-test

Hello, it would be greatly appreciated if you could update it

**Comment by aojea** [comment]
/lgtm
/approve

Thanks

**Pull request merged**